.github/workflows/v1.yml

```diff
@@ -0,0 +1,55 @@
+name: Run Tests
+
+on:
+  pull_request:
+    branches:
+      v1
+
+env:
+  GO111MODULE: on
+  GOPROXY: https://proxy.golang.org
```

Comment:
Any reason we're setting this explicitly?

The default is `https://proxy.golang.org,direct`, which seems better because we have a fallback if the proxy is down. From `go help modules`:

> The default setting for GOPROXY is "https://proxy.golang.org,direct", which means to try the
Go module mirror run by Google and fall back to a direct connection if the proxy reports that it does not have the module (HTTP error 404 or 410). See https://proxy.golang.org/privacy for the service's privacy policy. If GOPROXY is set to the string "direct", downloads use a direct connection to source control servers.

Reply:
It was a part of the old file which I simply carried over. I think @saschagrunert had added this and he would be the correct person to shed light on this.

Reply:
It speeds things up on go 1.12 and is the default for 1.13, so I think we can remove it but it should not make much differences.

Reply:
Just checked the `1.12` docs. Default on `1.12` is effectively `direct`, and `1.13` is `https://proxy.golang.org,direct`. 

I'm happy to keep it in to speed things up for pre-1.13, but I'd recommend changing it to include the `,direct` at the end for a fallback in case the proxy is down for whatever reason.

Reply:
Oops, 1.12 doesn't support comma separated lists. This is fine if we want to keep it, I'm going to resolve this.

Reply:
Alright!